patches/tModLoader/Terraria/ModLoader/VanillaDamageClasses.cs

```diff
@@ -47,6 +47,7 @@ public override StatInheritanceData GetModifierInheritance(DamageClass damageCla
 	}
 
 	public override bool GetEffectInheritance(DamageClass damageClass) => damageClass == Melee;
+	public override bool GetPrefixInheritance(DamageClass damageClass) => damageClass == Melee;
```

Comment:
Unnecessary?

Reply:
May have been left over from a design without any default inheritance, unnecessary and removed.